http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandlerAlgebra.scala

```diff
@@ -131,7 +136,16 @@ final case class PollStorageContainerParams(workspaceId: WorkspaceId, jobId: Wsm
 /**
  * This case class represents the necessary information to poll all objects associated with the runtime, namely disk and vm
  */
-final case class CreateRuntimeResourcesResult(vmRequest: CreateVmRequest, createDiskResult: CreateDiskForRuntimeResult)
+final case class CreateRuntimeResourcesResult(vmRequest: CreateVmRequest,
+                                              createDiskResult: CreateDiskForRuntimeResult,
+                                              hybridConnectionName: RelayHybridConnectionName,
+                                              primaryKey: PrimaryKey
```

Comment:
consider naming this something other than the type, which doesn't tell me what it is the primary key for in this context